### DISCUSSION TITLE
IOS15 make failed

### DISCUSSION THREAD

**Discussion by ocean-zhc** [theos/theos#609] [Questions]
1、$ make package install
2、How to open port 22 on iPhone IP？

==> Notice: Build may be slow as Theos isn’t using all available CPU cores on this computer. Consider upgrading GNU Make: https://github.com/theos/theos/wiki/Parallel-Building
> Making all for tweak demo…
make[2]: Nothing to be done for `internal-library-compile'.
> Making stage for tweak demo…
dm.pl: building package `com.ocean.demo:iphoneos-arm' in `./packages/com.ocean.demo_0.0.1-3+debug_iphoneos-arm.deb'
==> Installing…
ssh: connect to host 192.168.2.166 port 22: Connection refused
make: *** [internal-install] Error 255

3、
```
brew install usbmuxd
iproxy 4567 22
```
Map the 22 port of the iPhone to a certain port of the Mac via USB, for example: 4567, how to modify the default 22 port?

**Comment by kirb** [comment]
If you’re using iproxy, you’re forwarding a port on the phone connected over USB to a port on your PC. So it would be:

```bash
export THEOS_DEVICE_IP=localhost THEOS_DEVICE_PORT=4567
```

Though it sounds like you’re lost because you missed a step or two. If your device isn’t jailbroken, you need to do so first. Only iPhone X and older can be jailbroken on iOS 15 using [checkra1n](https://checkra.in). You then need to install OpenSSH Server (package id `openssh-server`).
